src/main/scala/org/broadinstitute/dsde/firecloud/model/ModelJsonProtocol.scala

```diff
@@ -325,4 +327,36 @@ object ModelJsonProtocol extends WorkspaceJsonSupport {
   implicit val impCreateRawlsBillingProjectFullRequestFormat = jsonFormat2(CreateRawlsBillingProjectFullRequest)
   implicit val impSpreadsheetResponse = jsonFormat1(SpreadsheetResponse)
 
+  implicit object ShareFormat extends RootJsonFormat[Share] {
```

Comment:
This could simply be `jsonFormat4(Share)`

You may need to `import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport.InstantFormat` to enable this.